### DISCUSSION TITLE
add additional 'sendActivationEmail' system setting instead of using 'requireEmailVerification' to trigger activation emails

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9438] [Ideas]
> Created by: Nate Iler (nate@flipboxdigital.com) on 2015/03/11 14:29:14 +0000
> Votes at time of UserVoice import: 2

-----------------------------------

It is currently impossible to suppress sending activation emails while requiring email verification.  These actions should be evaluated independently as in some instances new registrants should not be notified.  In addition, disabling admins from triggering notifications is also necessary.

**Comment by brandonkelly** [comment]
@nateiler You still feel this is needed?  If so what sort of flow would you hope for?

**Comment by engram-design** [comment]
@brandonkelly Just have a client asking for this. Our use-case is that from a customer/user point of view, they don't want them to have to activate their account if they themselves are the ones signing up. However, they do a lot of phone orders (Commerce site), and they want to be able to create users for phone orders, or in-person.

At this point, they want to be able to create the user on behalf of them, and send the user an activation email. I'm aware that this activation is technically not needed, as the user is already activated, but its a much better AX than creating the user and sending them a password reset email. That's the method at the moment, but its also a little awkward for the registered user (it'd be better to get an email prompting to activate, then reset a password to an account you suddenly have).

The reason we don't have this setting on at all is because we have a password field during checkout so that users can create an account then and there. Its annoying to go through checkout, only to get another email to activate your account, despite literally just buying something (and getting an order confirmation email).

Hope that makes sense? I guess I'm just proposing to take the `{% if requireEmailVerification %}` check out. Or at least have an additional check for `{% if requireEmailVerification or sendActivationEmail %}`.
